amplify/src/main/java/com/github/stkent/amplify/AmplifyView.java

```diff
@@ -215,14 +234,14 @@ public void onClick(final View v) {
                     askSecondQuestion();
                     break;
                 case POSITIVE:
+                    ratingStateTracker.notifyEventTriggered(IntegratedEvent.USER_GAVE_POSITIVE_FEEDBACK);
                     thankUser();
-//                    PlayStoreUtil.openPlayStoreToRate((Activity) getContext());
-//                    ratingStateTracker.notify(USER_GAVE_RATING);
+                    respondToPositiveFeedback();
                     break;
                 case NEGATIVE:
+                    ratingStateTracker.notifyEventTriggered(IntegratedEvent.USER_GAVE_NEGATIVE_FEEDBACK);
                     thankUser();
-//                    showFeedbackEmailChooser();
-//                    ratingStateTracker.notify(USER_GAVE_FEEDBACK);
+                    respondToNegativeFeedback();
```

Comment:
We should call a method on the state tracker here, like `checkAndShow`, and pass in an activity context.